gradle.properties

```diff
@@ -1,6 +1,6 @@
 group=io.ballerina.stdlib
 version=0.1.1-SNAPSHOT
```

Comment:
The version update should be the next minor version since we are updating the distribution version

Reply:
Fixed with [79ede356dc10294440a58f1335406a81ec58513b](https://github.com/ballerina-platform/module-ballerinax-activemq.driver/pull/8/commits/79ede356dc10294440a58f1335406a81ec58513b)